python/Ganga/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -72,10 +80,8 @@ def do_collective_operation(self, keep_going, method, *args, **kwds):
 
     def ids(self, minid=None, maxid=None):
         "Get the list of job ids. 'minid' and 'maxid' specify optional (inclusive) slice range."
-        if maxid is None:
-            maxid = sys.maxsize
-        if minid is None:
-            minid = 0
+        maxid = maxid if maxid not None else sys.maxsize
+        minid = minid if minid not None else 0
```

Comment:
@alexanderrichards firstly, the code above is invalid Python and should instead be:
```python
maxid = maxid if maxid is not None else sys.maxsize
```
and secondly, in a syntax-light language like Python, by the time that a statement is 50% keywords it gets very hard to read and keep the correct associations in your mind.

Ternary operators work well when they are branching on a simple boolean variable but combining them with equality checks means that too many things are happening on one line.